src/proto/mgmt/pool.proto

```diff
@@ -57,3 +57,19 @@ message PoolDestroyReq {
 message PoolDestroyResp {
 	int32 status = 1; // DAOS error code
 }
+
+
+message PoolListReq {
+	string lp_grp = 1;
```

Comment:
For these protobuf structures, the convention is longer names in camelcase, i.e.  "group", "numPools", no need for the variable name prefixes. The protobuf file generation will appropriately style the names depending on the language.

Also would be good to comment all of these parameters.